src/constants/config.json

```diff
@@ -5,10 +5,14 @@
     "dataLayerIndex": 1
   },
   "eduAPI": {
-    "authToken": "64aa60e488a555e2ef8b7f9d1621ec7e430dcb8b8baa0d00b8460eca5660d188",
+    "testing-only-authToken": "64aa60e488a555e2ef8b7f9d1621ec7e430dcb8b8baa0d00b8460eca5660d188",
```

Comment:
I just noticed that eduAPI's `authToken` variable isn't referenced anywhere else in the app - is this for future use or am I missing something? (e.g. implicit in one of our Panoptes libs.)